src/Kucoin/Futures/API/Candle.jl

```diff
@@ -0,0 +1,104 @@
+module Candle
+
+export CandleQuery,
+    CandleData,
+    candle
+
+using Serde
+using Dates, NanoDates, TimeZones
+
+using CryptoAPIs.Kucoin
+using CryptoAPIs.Kucoin: Data, Page
+using CryptoAPIs: Maybe, APIsRequest
+
+@enum TimeInterval m1 m5 m15 m30 h1 h2 h4 h8 h12 d1 w1
+
+Base.@kwdef struct CandleQuery <: KucoinPublicQuery
+    symbol::String
+    granularity::TimeInterval
+    from::Maybe{NanoDate} = nothing
+    to::Maybe{NanoDate} = nothing
+end
```

Comment:
```suggestion
Base.@kwdef struct CandleQuery <: KucoinPublicQuery
    symbol::String
    granularity::TimeInterval
    from::Maybe{DateTime} = nothing
    to::Maybe{DateTime} = nothing
end
```
In all query-structures, where a time type is required, we use `DateTime` type. 
This will need to be corrected in all methods in the current PR

Reply:
After discussion we decided to leave it as it is for now